lib/client/js/ui/controls/Menu.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Insert description of Menu class
+ *
+ * @private
+ * @class
+ * @returns {Menu}
+ */
+var Menu = (function () {
+
+    Menu.prototype = new Control();
```

Comment:
We need to define the Menu constructor that's internal to this IIFE (instantly invoked function expression). That-a-way, the `Menu` we return out of this function and set the global `Menu` to is a constructor.

Here's a constructor: `function Menu() {}`